### PR TITLE
Clarify error docs

### LINKED ISSUE #667: event property in TimeoutError
#### What would you like to discuss?

In the [source code](https://github.com/sindresorhus/got/blob/4a8722b8801851d60b8dbff0941e3f1714964df4/source/errors.js#L101) `TimeoutError` has `event` property, but it's missing in [documentation](https://github.com/sindresorhus/got#gottimeouterror). Can this property be considered as public?

Refs: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30790

#### Checklist

- [x] I have read the documentation.

### PR DESCRIPTION
**Pull request by szmarczak**
Clarify some error docs to avoid misunderstanding.

Fixes #667

#### Checklist

- [x] I have read the documentation.
- [x] I have included a pull request description of my changes.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: readme.md [hunk 1]
```diff
@@ -671,7 +671,7 @@ The default Got options.
 
 ## Errors
 
-Each error contains (if available) `body`, `statusCode`, `statusMessage`, `host`, `hostname`, `method`, `path`, `protocol`, `url`, and `gotOptions` properties to make debugging easier.
+Each error contains `host`, `hostname`, `method`, `path`, `protocol` and `url` properties to make debugging easier.
```
Comment (sindresorhus) [inline readme.md]: I don't think you meant to delete `gotOptions`.

Comment (szmarczak) [reply readme.md]: you're right, fixed :D

**Comment by clocked0ne** [comment]
@szmarczak @sindresorhus  I don't see how this is clearer given that `body` is now missing as a property in all cases? Why not leave the original indication at the top of the section and add the relevant "includes" that appear in each type of error?

**Comment by szmarczak** [comment]
> I don't see how this is clearer

Every single error has listed properties. There's no "guessing".

> body is now missing as a property in all cases

According to `body`: good spot, I just forgot to add it to docs :)
According to `property in all cases`: no it's not in all cases. Please check the source code.

**Comment by clocked0ne** [comment]
Okay, so apparently `body` is only on `HTTPError`, but it's still wrong and would be great to see that fixed :)

**Comment by szmarczak** [comment]
> it's still wrong and would be great to see that fixed :)

It is not wrong. It's always been like that. But if you want to request a feature, feel free to make an issue :)

**Pull request merged**